### ISSUE TITLE
Inline `[call]` nodes

### ISSUE THREAD

**Issue by michbarsinai** [IQSS/DataTaggingLibrary#182]
They optimizer will inline [call] nodes (i.e. replace the call node with the callee and its sub-tree). All [end] nodes in the callee subtree should be replaces with the [call] node's nextNode.

This will be done only for call nodes that are called from a single place.

The idea behind this is to allow easier coding, while still getting the hierarchy in the visualizations.

**Comment by michbarsinai** [comment]
Inline if:
1. callee node is called once and only once.
2. callee node is not part of the call node sub-tree.
